fe1-web/src/features/notification/screens/SingleNotificationScreen.tsx

```diff
@@ -0,0 +1,55 @@
+import { useNavigation, useRoute } from '@react-navigation/core';
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useMemo } from 'react';
+import { Text } from 'react-native';
+import { useStore } from 'react-redux';
+
+import ScreenWrapper from 'core/components/ScreenWrapper';
+
+import { NotificationHooks } from '../hooks';
+import { NotificationStackParamList } from '../navigation/NotificationStackParamList';
+import { getNotification } from '../reducer';
+
+type NavigationProps = StackScreenProps<NotificationStackParamList, 'Notification'>;
+
+const SingleNotificationScreen = () => {
+  const route = useRoute<NavigationProps['route']>();
+  const navigation = useNavigation<NavigationProps['navigation']>();
+  const { notificationId } = route.params;
+
+  const store = useStore();
+  const laoId = NotificationHooks.useCurrentLaoId();
+  const notification = getNotification(laoId.valueOf(), notificationId, store.getState());
```

Comment:
I find it weird to have to pass the redux storage to this function. As I commented earlier, it might be better to treat it as a selector instead.

Reply:
That is true and I fully agree that `useSelector` is more appropriate but since I thought we cannot use `createSelector` if no computation takes place, this was the only option left 😅